legal/satzung.md

```diff
@@ -0,0 +1,130 @@
+# Satzung
+
+
+## Name, Sitz, Geschäftsjahr
+
+1.   Der Verein trägt den Namen *open energy modeling foundation* (oemof)
+2.   Sitz des Vereins ist $(SITZ).
+3.   Das Geschäftsjahr ist das Kalenderjahr.
+
+
+## Zweck und Ziele
+
+1. Der Verein verfolgt ausschließlich und unmittelbar gemeinnützige Zwecke im Sinne des Abschnitts „Steuerbegünstigte Zwecke“ der Abgabenordnung.
+2. Zweck des Vereins ist die Förderung von Wissenschaft und Forschung und Förderung des Umweltschutzes.
```

Comment:
Bin mir tatsächlich nicht sicher, ob Umweltschutz als Vereinszweck zu uns passt. Unsere Tätigkeit muss sich mit dem decken, was das Finanzamt unter dem Vereinszweck versteht. Stattdessen etwas zur Bedeutung freier Software zur Energiesimulation für die Umwelt in eine Präambel schreiben?